src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -74,6 +53,21 @@ struct IterRangeComp {
 
 static constexpr char const * const         DocumentStatisticsEstimatorName("DocumentStatisticsEstimator");
 
+/////////////////////////////////////////////////////////////////////////
+///  \struct        FrequencyAndIndexStruct
+///  \brief         This struct is a combination of values of FrequencyMap and
+///                 IndexMap for the purpose of keys equivelence of two maps
+///
+struct FrequencyAndIndexStruct {
+    std::uint32_t const TermFrequency;                            // Words and the number of documents that it appears in
+    std::uint32_t const Index;
+
+    //When generating the DocumentStatisticsAnnotationData, it will call FrequencyAndIndexStruct copy constructor
+    //FEATURIZER_MOVE_CONSTRUCTOR_ONLY(FrequencyAndIndexStruct);
```

Comment:
>FEATURIZER_MOVE_CONSTRUCTOR_ONLY(FrequencyAndIndexStruct); [](start = 6, length = 58)

The code should not be copyable - let's figure out what is going on.

Reply:
solved